vagrant/provisioning/arkcase-ce-external-ldap-foia-pki.yml

```diff
@@ -20,6 +20,7 @@
     - pentaho-license
     - arkcase-prerequisites
     - arkcase-app
+    - start-arkcase
```

Comment:
foia role must run first before arkcase can be started, may want to always put start-arkcase as the very last role

Reply:
@kikomanaskov333 I guess you're going to push another commit restoring the changes you reverted, with start-arkcase at the end of each play?

Reply:
Yes @david-oc-miller , now i'm going to push right commit with corrected plays.